### ISSUE TITLE
rocker/verse 4.3.1 fails base tests

### ISSUE THREAD

**Issue by bergsmat** [rocker-org/rocker-versioned2#715]
### Container image name

rocker/verse:4.3.1

### Container image digest

rocker/verse:4.3.1@sha256:89ce46ede9a2ae243f74a7d203837ccccef88fdb21ae31830b55c58a91216d5e

### What operating system are you seeing the problem on?

Windows

### System information

- Docker version v20.10.12
- Windows version 10.0.190144

### Bug description

I have a docker file consisting of the single line:
```FROM rocker/verse:4.3.1```

After creating a container from the resulting image, I execute this at the Rstudio console prompt:

```
Sys.setenv(LC_COLLATE = "C", LC_TIME = "C", LANGUAGE = "en")
tools::testInstalledPackages(scope = "base")
```
per advice [here.](https://rstudio.github.io/r-manuals/r-admin/Installing-R-under-Windows.html#testing-an-installation)

My result is:

```
Testing examples for package ‘base’
Testing examples for package ‘tools’
Testing examples for package ‘utils’
Running vignettes for package ‘utils’
  Running ‘Sweave.Rnw’

*** Source Errors ***

File Sweave.R:
Running 'texi2dvi' on 'example-1.tex' failed.
LaTeX errors:
! LaTeX Error: File `fancyvrb.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
<read *> 
         
l.26 \begingroup
                ^^M
!  ==> Fatal error occurred, no output PDF file produced!

*** Weave Errors ***

File Sweave.Rnw:
 chunk 2 
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'example-1.tex' failed.
LaTeX errors:
! LaTeX Error: File `fancyvrb.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
<read *> 
         
l.26 \begingroup
                ^^M
!  ==> Fatal error occurred, no output PDF file produced!

Testing examples for package ‘grDevices’
Testing examples for package ‘graphics’
...
Testing examples for package ‘splines’
Testing examples for package ‘stats4’
Testing examples for package ‘tcltk’
Error: testing 'tcltk' failed
```

In summary, ```fancyvrb``` not found, and ```tcltk``` failed for undiagnosed reasons.  Unexpected, yes?

Many thanks.

### How to reproduce this bug?

* create docker image using "FROM rocker/verse:4.3.1"
* create a container from resulting image
* execute at Rstudio command prompt: Sys.setenv(LC_COLLATE = "C", LC_TIME = "C", LANGUAGE = "en")
* execute at Rstudio command prompt: tools::testInstalledPackages(scope = "base")

**Comment by eitsupi** [comment]
Thanks for your report.

> `tcltk` failed for undiagnosed reasons. Unexpected, yes?

This is expected behavior. See #268.

**Comment by bergsmat** [comment]
I can live with the tcltk error, thanks.  Any thoughts about the ```utils``` error?

**Comment by eddelbuettel** [comment]
It's a fairly standard matter of part of R expecting more obscure parts of TeXLive.  If you install the required style it will render.  I would consider this to be optional behavior.

**Comment by bergsmat** [comment]
I will proceed accordingly, thanks!
